firebase-firestore/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Unreleased
 
+# 17.1.5
+- [changed] Firestore now recovers more quickly from bad network states.
+- [changed] Performance optimizations to speed up reading large collections.
+- [fixed] Offline persistence now properly records schema downgrades. This is a
+  forward-looking change that allows versions past this one to safely downgrade
+  to this version. Downgrading to versions prior to this one can be safe
+  depending on the source version. For example, downgrading from 17.1.4 to
+  17.1.2 is safe because there have been no schema changes between these
+  releases.  Related:
```

Comment:
Ultra-nit: in the iOS repo we've been referring to the issue number in parens, like so:

```
[fixed] Fixed an issue where gRPC would crash if shut down multiple times (#2146)
```

Reply:
Done. I've left them in the "official" release notes (http://cl/225058690) since those don't otherwise have any reference to this github project.